src/extra/helpers/helpers.hpp

```diff
@@ -0,0 +1,90 @@
+#ifndef C55FCF15_9B8F_44F0_AA7B_73BA2144579D
+#define C55FCF15_9B8F_44F0_AA7B_73BA2144579D
+
+#include "../gfx.hpp"
+#include <common_types.hpp>
+#include <foundation.hpp>
+#include <gfx/fwd.hpp>
+#include <gfx/helpers/gizmos.hpp>
+#include <gfx/helpers/wireframe.hpp>
+
+namespace shards {
+// Shards for rendering visual helpers
+// Gizmos, grids, lines, etc.
+namespace Helpers {
+
+struct Context {
+  static constexpr uint32_t TypeId = 'HLPc';
+  static inline shards::Type Type{{SHType::Object, {.object = {.vendorId = gfx::VendorId, .typeId = TypeId}}}};
```

Comment:
Could probably use the new helper `Type::Object`.
```suggestion
  static inline shards::Type Type::Object(gfx::VendorId, TypeId);
```